search-service/src/test/resources/ngsild/csr/contextSourceRegistration_minimal_entities.json

```diff
@@ -6,7 +6,7 @@
       "entities": [
         {
           "id": "urn:ngsi-ld:Vehicle:A456",
-          "type": "Vehicle"
+          "type": "BeeHive"
```

Comment:
weird

Reply:
Since i am filtering on the type in ContextSourceRegistrationServiceTest (line 294). i needed a type that was already defined in the JsonLdContextUtils file

Reply:
so align the entity id, this kind of misalignment will for sure cause headaches in the future